source/matplot/backend/gnuplot.h

```diff
@@ -47,7 +47,7 @@ namespace matplot::backend {
         /// Identify the default terminal type in the system
         static std::string default_terminal_type();
         static bool terminal_is_available(std::string_view);
-        static std::tuple<int, int, int> gnuplot_version();
+        static Version gnuplot_version();
```

Comment:
So converting the output to a `tuple` wouldn't break the API? A `Version` is a better decision than a `tuple,` but it's hard to tell how many people are counting on this API.

Reply:
Right, I have reverted `Version` to `std::tuple` as I did not know that one can compare them in such elegant way. I used to kill it if the code is not about generic programming (Scott Meyers book).
I learned something today, thanks!